src/components/AvatarGroup.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { AvatarGroup as PasteAvatarGroup } from "@twilio-paste/avatar";
+import Avatar from "./Avatar";
+import { IconSize } from "@twilio-paste/style-props";
+
+type AvatarGroupProps = {
+  names: string[];
+  size?: IconSize;
+};
+
+const AvatarGroup: React.FC<AvatarGroupProps> = ({ names, size }) => {
```

Comment:
I'm not sure if we even need this as a reusable component, but decided to leave it in.